disnake/ext/commands/slash_core.py

```diff
@@ -299,6 +310,15 @@ def __init__(
                 "Cannot set `default_member_permissions` or `dm_permission` on subcommands"
             )
 
+    @property
+    def root_command(self) -> InvokableSlashCommand:
+        """:class:`InvokableSlashCommand`: Returns the root command containing this subcommand,
+        even if the parent is an instance of :class:`SubCommandGroup`.
+
+        .. versionadded:: 2.6
+        """
+        return self.parent.parent if isinstance(self.parent, SubCommandGroup) else self.parent
+
```

Comment:
I think for typing, we should have `root_command` defined `SubCommandGroup` as well.

That way every option and subcommand of a slash command has a `root_command` attribute which makes typing easier.

Reply:
I actually wanted to hear that, let me implement this rq